tools/make/golang.mk

```diff
@@ -48,7 +48,10 @@ go.test.unit: ## Run go unit tests
 go.testdata.complete: ## Override test ouputdata
 	go test -timeout 30s -run ^TestTranslateXds github.com/envoyproxy/gateway/internal/xds/translator --override-testdata=true
 	go test -timeout 30s -run ^TestTranslate github.com/envoyproxy/gateway/internal/cmd/egctl --override-testdata=true
-
+	go test -timeout 30s -run ^TestExtractSubResourcesConfigDump github.com/envoyproxy/gateway/internal/cmd/egctl --override-testdata=true
```

Comment:
this makes the target faster but also requires the developer to remember to add the Test name in here, can we just run this on all tests ?

Reply:
Yep, that is right.